backend/db/utils.py

```diff
@@ -282,11 +287,27 @@ def is_refresh_active(local=False) -> bool:
     determined that this is in error and the refresh is considered inactive. 6 hours was chosen because this is the
     default maximum amount of time that a GitHub action can run, but it is also well over the normal amount of time that
     the refresh takes."""
-    last_start = dp.parse(check_db_status_var('last_refresh_request', local))
-    last_end = dp.parse(check_db_status_var('last_refresh_exited', local))
-    considered_active = last_start > last_end
-    hours_since_last_refresh: float = (dp.parse(current_datetime()) - last_end).total_seconds() / 60 / 60
-    return considered_active and hours_since_last_refresh < 6
+    # Always check if derived tables refresh active for any case. Else check more for standard refresh status
+    key_pairs = [('last_derived_refresh_request', 'last_derived_refresh_exited')]
+    if refresh_type == 'standard':
+        key_pairs = [('last_refresh_request', 'last_refresh_exited')] + key_pairs
```

Comment:
## Normal refresh now won't start if derived tables are active
I changed this behavior mainly handle any unanticipated edge cases that might occur. Basically, the way it was before was that if the refresh was currently active, but the derived tables refresh were active for some reason (e.g. vocab/counts refresh or some other operation in local development), the refresh would start, and it would get to the point where it start to do the derived table refresh. Then, if it detected that the derived tables were currently being updated, it would wait up to two hours. Perhaps it was fine to keep it this way, but our refresh runs every 20 minutes, so I think it's fine to simply not start the refresh at all if the derived tables are running.